numerics/fast_sin_cos_cycle.cpp

```diff
@@ -0,0 +1,62 @@
+
+#pragma once
+
+#include "numerics/fast_sin_cos_cycle.hpp"
+
+#include <pmmintrin.h>
+
+#include "base/macros.hpp"
+#include "numerics/polynomial.hpp"
+#include "numerics/polynomial_evaluators.hpp"
+
+namespace principia {
+namespace numerics {
+
+namespace {
+
+using P3 = PolynomialInMonomialBasis</*Value=*/double,
+                                     /*Argument=*/double,
+                                     /*degree=*/3,
+                                     /*Evaluator=*/EstrinEvaluator>;
+P3 sin_polynomial(P3::Coefficients{6.28316404405113818577981340506,
```

Comment:
Document the polynomials, i.e, something like "the \<k>th order polynomial that minimizes \<error function> on \<function> under \<constraints> on \<domain>".